packages/expo-updates/ios/EXUpdates/AppController.swift

```diff
@@ -328,6 +334,68 @@ public class AppController: NSObject, AppLoaderTaskDelegate, ErrorRecoveryDelega
     return launcher?.launchedUpdate
   }
 
+  // MARK: - Notifications for checkForUpdate
```

Comment:
not sure how useful this particular event will be to gather state, but I guess it could be useful if one of the state fields is something like `isCheckingForUpdate`.